### PR TITLE
Add note on KUBE_TIMEOUT to resolve timeout panic

### PR DESCRIPTION
**Pull request by johnarok**
For #1594

### CODE REVIEW

File: contributors/devel/testing.md [hunk 1]
```diff
@@ -54,6 +54,12 @@ cd kubernetes
 make test  # Run all unit tests.
 ```
 
+if any unit test fails with a timeout panic (see #1594) on the testing package. you can increase the KUBE_TIMEOUT value as shown below.
```
Comment (cblecker) [inline contributors/devel/testing.md]: "If" and "You" should be capitalized at the beginning of sentences.

Comment (cblecker) [reply contributors/devel/testing.md]: Can you add a link to the issue number?
Something like this:
```
(see [#1594](https://github.com/kubernetes/community/issues/1594))
```

Comment (cblecker) [reply contributors/devel/testing.md]: Please wrap `KUBE_TIMEOUT` in single backticks.

Comment (johnarok) [reply contributors/devel/testing.md]: Thank you for the review, I have made the changes.

**Pull request merged**